package.json

```diff
@@ -63,14 +64,14 @@
     "sass-loader": "^7.1.0",
     "semantic-release": "^17.0.4",
     "terser-webpack-plugin": "^1.2.3",
+    "typescript": "^4.0.3",
```

Comment:
Sorry, I've been moving. I am fairly certain it is, for testing. I would have tried to avoid adding it otherwise.

Unfortunately, I don't remember exactly what required it. In this commit, I updated @vue/test-utils to 2.0.*.beta and removed vue-template-compiler. My guess/pseudo-memory is there is something to do with template compilation test-utils is doing in the new version and it uses typescript somehow? I'll see if I can figure it out tomorrow.

Reply:
Ah, it's vue-jest actually. Without typescript installed, tests complain with e.g.:

```
 FAIL  tests/unit/OptionComparator.spec.js
  ● Test suite failed to run

    Unable to load the module "typescript". Using "ts-jest" requires this package to be installed. To fix it:
        ↳ install "typescript": `npm i -D typescript` (or `yarn add --dev typescript`)

      at Importer._import (node_modules/ts-jest/dist/util/importer.js:143:15)
      at Importer.typescript (node_modules/ts-jest/dist/util/importer.js:74:21)
      at ConfigSet.get (node_modules/ts-jest/dist/config/config-set.js:384:40)
      at ConfigSet.compilerModule (node_modules/ts-jest/dist/util/memoize.js:43:24)
      at ConfigSet.readTsConfig (node_modules/ts-jest/dist/config/config-set.js:625:23)
      at ConfigSet.get (node_modules/ts-jest/dist/config/config-set.js:313:31)
      at ConfigSet._typescript (node_modules/ts-jest/dist/util/memoize.js:43:24)
      at ConfigSet.get [as typescript] (node_modules/ts-jest/dist/config/config-set.js:283:25)
      at getTsJestConfig (node_modules/vue-jest/lib/utils.js:70:30)
      at processTemplate (node_modules/vue-jest/lib/process.js:100:20)
```

But updating `vue-jest` from `5.0.0-alpha.5` to `5.0.0-alpha.7` fixes this. Must have been configuration problem with that version. Should be able to remove typescript by updating vue-jest!

Reply:
Fixed in https://github.com/sagalbot/vue-select/pull/1377. Updated `vue-jest` to `5.0.0-alpha.8` and removed `typescript`.